database/migrations/2019_07_26_192824_make_recurring_earnings.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class MakeRecurringEarnings extends Migration {
+    public function up() {
+        Schema::table('earnings', function (Blueprint $table) {
+            $table->integer('recurring_id')->unsigned()->nullable();
+            $table->integer('tag_id')->unsigned()->nullable();
+            $table->foreign('recurring_id')->references('id')->on('recurrings')->onDelete('cascade');
+            $table->foreign('tag_id')->references('id')->on('tags')->onDelete('cascade');
+        });
+
+        Schema::table('recurrings', function (Blueprint $table) {
+            $table->enum('transaction_type', ['earning', 'spending']);
+        });
+    }
+
+    public function down()
+    {
```

Comment:
Same goes here, please put the opening bracket on the same line <3

Reply:
Well. This is the Laravel template when I create a new migration-file, but will fix it